lightning/src/ln/channel.rs

```diff
@@ -1726,8 +1727,13 @@ pub(super) trait InteractivelyFunded<SP: Deref> where SP::Target: SignerProvider
 
 	fn tx_complete(&mut self, msg: &msgs::TxComplete) -> HandleTxCompleteResult {
 		HandleTxCompleteResult(match self.interactive_tx_constructor_mut() {
-			Some(ref mut tx_constructor) => tx_constructor.handle_tx_complete(msg).map_err(
-				|reason| reason.into_tx_abort_msg(self.context().channel_id())),
+			Some(ref mut tx_constructor) => tx_constructor.handle_tx_complete(msg)
+				.inspect(|tx_complete_value| {
+					if let HandleTxCompleteValue::SendTxComplete(_, signing_session) = tx_complete_value {
+						self.context_mut().next_funding_txid = Some(signing_session.unsigned_tx.txid());
+					}
+				})
```

Comment:
Can't used `inspect` as it hasn't made it into our MSRV (1.63.0).

Reply:
Yeah I've removed it locally and haven't pushed latest changes yet :+1: